src/graphql/blocks.graphql

```diff
@@ -4,11 +4,7 @@ subscription BlocksListener($limit: Int = 7, $offset: Int = 0) {
     proposer
     hash
     timestamp
-    transactionsAggregate: transactions_aggregate {
-      aggregate {
-        count
-      }
-    }
+    transactionsAggregate: num_txs
```

Comment:
This will break the hooks folder. Please fix fix

Reply:
Fixed.